common.blocks/menu/menu.js

```diff
@@ -37,6 +37,19 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends menu.
             this.getItems().forEach(function(menuItem){
                 menuItem.setMod(modName, modVal);
             });
+            this.domElem.attr('aria-disabled', !!modVal);
+        },
+
+        'focused' : {
+            'true' : function() {
+                this.__base.apply(this, arguments);
+                this._activedescendant || (this._activedescendant = identify());
```

Comment:
1) It looks like the code relayed on `_activedescendant` won't work properly if `menu` is initialised with `focused` modifier set.
2) As there can't be several elements on the page with `focused` modifier (we can't have several focused buttons simultaneously), it looks redundant to call `identify()` for every menu on the page. Can't we have it as a local constant? E.g. 
```js
var CURRENT_ACTIVE_DESCENDAT = identify()
// ···
onSetMod : {
  focused : {
    true : function() {
      this.domElem.attr('aria-activedescendant', this._activeDescendant = CURRENT_ACTIVE_DESCENDAT);
    },
    '' : function() {
      this.domElem.removeAttr('aria-activedescendant');
      this._activeDescendant = null;
    }
```